harper-dictionary-parsing/src/lib.rs

```diff
@@ -1,24 +1,30 @@
 mod affix_replacement;
```

Comment:
Would you mind writing up a `README.md` for this crate? If so, make sure you link it in this file the same way they are for the other crates.